src/lib/dataEntities/networks.ts

```diff
@@ -28,14 +28,16 @@ export interface L1Network extends Network {
 }
 
 export interface L2Network extends Network {
+  partnerChainIDs: number[]
   tokenBridge: TokenBridge
   ethBridge: EthBridge
   partnerChainID: number
-  isArbitrum: true
+  isArbitrum: boolean
```

Comment:
I think we can keep this as `true`, no?

Reply:
Do we want Orbit chains to be flagged as Arbitrum tho?

Reply:
Well they are all Arbitrum chains, so I would say yes. Not sure if that messes up anything, could you check?